migrations/20191002105645-create-user.js

```diff
@@ -4,19 +4,40 @@ module.exports = {
     id: {
       allowNull: false,
       autoIncrement: true,
-      primaryKey: true,
       type: Sequelize.INTEGER,
     },
     firstName: {
       type: Sequelize.STRING,
     },
+    middleName: {
+      type: Sequelize.STRING,
+    },
     lastName: {
       type: Sequelize.STRING,
     },
+    role: {
+      type: Sequelize.STRING,
+      defaultValue: 'member',
+    },
+    membershipNo: {
```

Comment:
I think the id can serve as the membershipNo. Since new member joining the cooperative will have to signup using the app and not that the details will be manually entered by an admin. @omorilewa what do you think?

Reply:
The id field should be sequential but the membershipNo may not be sequential and may be tweaked in the future.